src/applications/personalization/profile360/components/ProfileView.jsx

```diff
@@ -112,6 +114,12 @@ class ProfileView extends React.Component {
             ]}
           >
             <div>
+              {/* Breadcrumbs */}
+              <Breadcrumbs className="medium-screen:vads-u-padding-y--2">
+                <a href="/">Home</a>
+                <a href="/">Your profile</a>
```

Comment:
I'm rendering BreadCrumbs here instead of in `content` since it appears we should render them regardless of the user status.